internal/config/plugin.go

```diff
@@ -73,6 +73,23 @@ func (p Plugins) Load(path string, loadExtra bool) error {
 	// Load from XDG dirs
 	const k9sPluginsDir = "k9s/plugins"
 	for _, dir := range append(xdg.DataDirs, xdg.DataHome, xdg.ConfigHome) {
+		// Try to access the XDG directory. If it's not accessible, either because
```

Comment:
I think this only solves the issue at the top level but what if we have a/b/c where user has rights to a but not b? 
Perhaps best to add a helper to check existence and rights during the recursive descent and catch all potential issues?